pkgs/misc/vim-plugins/update.py

```diff
@@ -450,6 +453,13 @@ def parse_args():
             f"By default from {DEFAULT_IN} to {DEFAULT_OUT}"
         )
     )
+    parser.add_argument(
+        "--add",
+        "-a",
```

Comment:
I'd skip the short option here, since it shouldn't be a common operation and maybe we want to use `-a` for something else in the future. I don't feel very strongly about this though if you disagree.